scripts/_common.sh

```diff
@@ -86,6 +86,74 @@ myynh_add_swap() {
 		fi
 }
 
+# Install libheif and libvips from source for HEIC support
+# Based on https://github.com/community-scripts/ProxmoxVE/blob/main/install/immich-install.sh
+# and https://github.com/immich-app/base-images/blob/main/server/Dockerfile
+myynh_install_libvips() {
+	local build_dir="$source_dir/vips-build"
+	local libs_dir="$install_dir/vips"
+
+	# https://github.com/immich-app/base-images/blob/main/server/sources/libheif.json
+	local libheif_version="1.20.2"
+	# https://github.com/immich-app/base-images/blob/main/server/sources/libvips.json
+	local libvips_version="8.17.3"
+
+	mkdir -p "$build_dir" "$libs_dir"
+	pushd "$build_dir"
+
+	# Build libheif
+	ynh_print_info "Building libheif $libheif_version for HEIC support..."
+	ynh_hide_warnings git clone --depth 1 --branch "v$libheif_version" https://github.com/strukturag/libheif.git
+	pushd libheif
+
+	mkdir -p build
+	cd build
+	ynh_hide_warnings cmake --preset=release-noplugins \
+		-DCMAKE_INSTALL_PREFIX="$libs_dir" \
+		-DWITH_DAV1D=ON \
+		-DENABLE_PARALLEL_TILE_DECODING=ON \
+		-DWITH_LIBSHARPYUV=ON \
+		-DWITH_LIBDE265=ON \
+		-DWITH_AOM_DECODER=OFF \
+		-DWITH_AOM_ENCODER=ON \
+		-DWITH_X265=OFF \
+		-DWITH_EXAMPLES=OFF \
+		..
+	ynh_hide_warnings make -j "$(nproc)"
+	ynh_hide_warnings make install
+	popd
+
+	# Build libvips
+	ynh_print_info "Building libvips $libvips_version with HEIC support..."
+	export PKG_CONFIG_PATH="$libs_dir/lib/pkgconfig:${PKG_CONFIG_PATH:-}"
+	export LD_LIBRARY_PATH="$libs_dir/lib:${LD_LIBRARY_PATH:-}"
+	ynh_hide_warnings git clone --depth 1 --branch "v$libvips_version" https://github.com/libvips/libvips.git
+	pushd libvips
+
+	ynh_hide_warnings meson setup build --buildtype=release \
+		--prefix="$libs_dir" \
+		--libdir=lib \
+		-Dintrospection=disabled \
+		-Dtiff=disabled
+	cd build
+	ynh_hide_warnings ninja install
+	popd
+
+	# Return to original directory
+	popd
+
+	# Cleanup
+	ynh_print_info "Cleaning up libvips build directory..."
+	ynh_secure_remove "$build_dir"
```

Comment:
```suggestion
	ynh_safe_rm "$build_dir"
```

The helper has changed with version 2.1